### DISCUSSION TITLE
Can I sort the blogposts, so that the last post is on the top?

### DISCUSSION THREAD

**Discussion by BogusCurry** [datenstrom/community#227] [Ask a question] [answered]
Hello there ;D

I have make a blogpost, but the post is not on the top. How I can this make, so that the last post is on the top ?
Here my blogsite ;D
http://www.pinguinsreisen.de/edit/blog/

Have a nice Day ;D
Bogus

**Comment by schulle4u** [comment, marked as answer]
Looks like the date of the post is wrong. The first entry I can see is from august 1th, 2017. Check the head section in your blog pages, usually dates are formated as `yyyy-mm-dd`, e.g. january 22th 2017 would be `2017-01-22`

HTH
Steffen
